test/libdl.jl

```diff
@@ -25,9 +25,10 @@ cd(dirname(@__FILE__)) do
 # @test !isempty(Libdl.find_library(["libccalltest"], [dirname(@__FILE__)]))
 
 # dlopen should be able to handle absolute and relative paths, with and without dlext
+private_libdir = dirname(abspath(Libdl.dlpath("libjulia")))
```

Comment:
this will cause julia-debug to segfault (because libjulia gets loaded at the same time). a better option might be `dirname(bytestring(Base.JLOptions().image_file))`, although you might have to adjust the install directory for libccalltest